modules/mri_upload/php/NDB_Menu_Filter_mri_upload.class.inc

```diff
@@ -238,39 +249,42 @@ class NDB_Menu_Filter_Mri_Upload extends NDB_Menu_Filter_Form
         $file_name = $file->fileInfo['name'];
         $file_path = $temp_dir. "/". $file_name;
         ///////////////////////////////////////////////////////////////////////
-        ////////////// make sure the advanced options are filled out///////////
-        ///////////////////////////////////////////////////////////////////////
-        $candid      = trim($args['values']['CandID']);
-        $pscid       = trim($args['values']['PSCID']);
-        $visit_label = trim($args['values']['VisitLabel']);
-
-        if ((empty($candid)) || (empty($pscid)) || (empty($visit_label))) {
-            $errors[] ="Make sure the CandID, PSCID and Visit_label are".
-            "filled out ";
-        }
-
-        $patient_name = $pscid . "_" . $candid . "_" . $visit_label;
-        ///////////////////////////////////////////////////////////////////////
-        //////check to see if candid/pscid and visit-label exist///////////////
+        /////////Validate the advanced Options only if it's not a Phantom//////
         ///////////////////////////////////////////////////////////////////////
-        if ((!empty($candid)) && (!empty($pscid)) && (!empty($visit_label))) {
-            if ($db->pselectOne(
-                "SELECT COUNT(*) FROM session s
-                 JOIN candidate c ON (c.candid=s.candid)
-                 WHERE s.Visit_label = :vlabel
-                 AND c.pscid = :pid AND c.candid = :cid",
-                array(
-                 'cid'    => $candid,
-                 'vlabel' => $visit_label,
-                 'pid'    => $pscid,
-                )
-            )==0) {
-                    $errors[] = "Make sure you enter the CandID: $candid ,".
-                                "PSCID: $pscid and the corresponding Visit-label: " .
-                                "$visit_label which already exist";
+        if ($args['values']['IsPhantom']=='N') {
+            ///////////////////////////////////////////////////////////////////////
+            ////////////// make sure the advanced options are filled out///////////
+            ///////////////////////////////////////////////////////////////////////
+            $candid      = trim($args['values']['CandID']);
+            $pscid       = trim($args['values']['PSCID']);
+            $visit_label = trim($args['values']['VisitLabel']);
+            if ((empty($candid)) || (empty($pscid)) || (empty($visit_label))) {
+                $errors[] ="Make sure the CandID, PSCID and Visit_label are ".
+                "filled out ";
+            }
+            $patient_name = $pscid . "_" . $candid . "_" . $visit_label;
```

Comment:
This variable doesn't appear to be used.. maybe run phpmd to check for other unused variables?